code/game/dna/dna3.dm

```diff
@@ -0,0 +1,18 @@
+//DNA 3: Revenge of The Geneticists
+
+/datum/dna
+	var/list/mutations
+
+/datum/dna/New()
+	mutations = list()
+	add_random_mutations(1, 3)
+
+/datum/dna/proc/add_random_mutations(var/min, var/max)
```

Comment:
We don't do `var/` in proc definition. Those aren't of any use, they just take space and make the code more cluttered.